server.py

```diff
@@ -1,7 +1,90 @@
-from flask import Blueprint, Flask, render_template, request
+import os
+import hashlib
 import sqlite3
+from flask_sqlalchemy import SQLAlchemy
+from flask_login import LoginManager, UserMixin, login_user, logout_user, current_user
+from flask import Flask, render_template, request, redirect, url_for
 app = Flask(__name__)
 
+# Secret key of random bytes (used for sessions and user login implementation)
+app.secret_key = b'\x15\xbe\x9bW\x80\xa1\x8d\xe0\x907\x08D\xfbr\x81\xf7'
```

Comment:
Do we want a secret key directly in the code?

Reply:
Make it random and I think this is good